### DISCUSSION TITLE
libtirpc-devel on RHEL-9

### DISCUSSION THREAD

**Discussion by jabrown893** [libMesh/libmesh#3863] [General]
It seems that libtirpc-devel is no longer available as a yum installation on RHEL-9. How/where should I get the xdr/rpc files needed to compile with xdr support?

Thank you!

**Comment by jwpeterson** [comment]
Sorry, I don't know the answer to this. Have you tried asking the RHEL-9 maintainers about this? If you find out anything, please update the discussion here since it will likely be useful to others. And, if RHEL is dropping the rpc/xdr stuff entirely, that would be good for us to know. I would have thought it was something so low-level and small that it would stick around forever, but given the way that it has slowly been moved out of /usr/include, it's possible that it's being removed entirely...

**Comment by roystgnr** [comment]
Are you sure you have every repository you need enabled?  I'm seeing libtirpc-devel listed in RHEL9 docs [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/package_manifest/index#CodeReadyLinuxBuilder-repository) under the introduction,

> The CodeReady Linux Builder repository contains additional packages for use by developers. This repository is available with all Red Hat Enterprise Linux subscriptions but does not need to be enabled on your runtime deployments.

Though this is worrisome:

> Packages included in the CodeReady Linux Builder repository are unsupported.
